File: graphql-api/src/main/java/io/prizy/publicapi/port/dto/AppConfigurationDto.java
package io.prizy.publicapi.port.dto;

import lombok.Builder;

/**
 * @author Nidhal Dogga
 * @created 6/24/2022 9:26 PM
 */


@Builder
public record AppConfigurationDto(
  String usageConditionsUrl,
  String minimumAppVersion,
  String latestAppVersion,
  String privacyPolicyUrl,
  String legalNoticeUrl
) {

}

File: public-api/src/main/kotlin/io/prizy/publicapi/port/app/dto/AppConfigurationDto.kt
package io.prizy.publicapi.port.app.dto

import com.expediagroup.graphql.generator.annotations.GraphQLName

/**
 *  @author Nidhal Dogga
 *  @created 5/3/2022 12:15 AM
 */

@GraphQLName("AppConfiguration")
data class AppConfigurationDto(
  val usageConditionsUrl: String,
  val minimumAppVersion: String,
  val latestAppVersion: String,
  val privacyPolicyUrl: String,
  val legalNoticeUrl: String
)
